File: TVTrackerServer/TVTracker.Entity/Entity/Models/WatchedEpisode.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TVTracker.Entity.Entity.Models
{
	public class WatchedEpisode : Entity
	{
		public int userId { get; set; }

		public int episodeId { get; set; }
	}
}

File: TVTracker/app/src/main/java/com/tvtracker/models/WatchedEpisode.java
package com.tvtracker.models;

public class WatchedEpisode {
    public int userId;
    public int id;
    public Boolean watched;

    public WatchedEpisode(int userId, int id, Boolean watched) {
        this.userId = userId;
        this.id = id;
        this.watched = watched;
    }
}
